android/res/layout/fragment_editor.xml

```diff
@@ -35,7 +35,8 @@
             android:id="@+id/input"
             style="@style/MwmWidget.Editor.FieldLayout.EditText"
             android:hint="@string/editor_edit_place_name_hint"
-            android:singleLine="true"/>
+            android:singleLine="true"
+            android:theme="@style/MwmTheme.Editor.EditText"/>
```

Comment:
Напрямую не получится, разве что в стиле установить тему… Выглядит извратом каким-то, но можно)

> On Mar 21, 2016, at 4:39 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/fragment_editor.xml <https://github.com/mapsme/omim/pull/2377#discussion_r56822414>:
> 
> > @@ -35,7 +35,8 @@
> >              android:id="@+id/input"
> >              style="@style/MwmWidget.Editor.FieldLayout.EditText"
> >              android:hint="@string/editor_edit_place_name_hint"
> > -            android:singleLine="true"/>
> > +            android:singleLine="true"
> > +            android:theme="@style/MwmTheme.Editor.EditText"/>
> Тогда может положить атрибут в стиль?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2377/files/20f907b1c8a91056bcb53c40688f703dcee9dcf8#r56822414>